src/meta/src/model/stream.rs

```diff
@@ -74,6 +75,13 @@ impl StreamEnvironment {
         }
     }
 
```

Comment:
(Should comment on L71)

Shall we rename `StreamEnvironment` to `StreamContext`, etc.? There is an `StreamEnvironment` in the `stream` crate and might be confusing.

Reply:
By the way, propose a new `ExprContext` as a substitude for `CapturedExecutionContext` because the `CapturedContext` is designed for expression (it's in the `expr` crate IIRC)